website/src/pages/login.page.tsx

```diff
@@ -0,0 +1,176 @@
+  form,
+  name,
+  label,
+  type,
+  placeholder,
+}: FormFieldsType) => {
+  return (
+    <>
+      <FormLabel {...form} name={name} label={label} />
+
+      <FormInput
+        {...form}
+        name={name}
+        className={loginFormBox}
+        type={type}
+        placeholder={placeholder}
+      />
+    </>
+  )
+}
+
+const LoginPage = () => {
+  const { loginUser } = useUser().operations
+
+  const loginForm = useFormState({
+    values: { username: "", password: "" },
+    onValidate: (values) => {
+      if (!values.username || !values.password) {
+        throw (
+          (!values.username && { username: "A username is required" }) || {
+            password: "A password is required",
+          }
+        )
+      }
+    },
+    onSubmit: (values) => {
+      loginUser(values.username, values.password)
+    },
+  })
+
+  return (
+    <Layout>
+      <main className={skinnyWidth}>
+        <header>
+          <h1 className={centeredColumn}>Login to Your Account</h1>
+          <h4>
+            Login to contribute to the archive by transcribing documents,
+            recording pronunciations, providing cultural commentary, and more.
+          </h4>
+        </header>
+
+        <Form {...loginForm} className={centeredForm}>
+          <FormFields
+            form={loginForm}
+            name="username"
+            label="Email *"
+            placeholder="mail@website.com"
+          />
+
+          <FormFields
+            form={loginForm}
+            name="password"
+            label="Password *"
+            type="password"
+            placeholder="enter password"
+          />
+
+          <FormMessage {...loginForm} name="username" />
+          <FormMessage {...loginForm} name="password" />
+
+          <ResetLink />
+
+          <div className={positionButton}>
+            <FormSubmitButton {...loginForm} className={submitButton}>
+              Log in
+            </FormSubmitButton>
+          </div>
+        </Form>
+      </main>
+    </Layout>
+  )
+}
+
+// the login button that appears in the header of the website
+export const LoginHeaderButton = () => {
+  const { user } = useUser()
+  const { logoutUser } = useUser().operations
+
+  return (
+    <>
+      <div className={loginHeader}>
+        {/* show a logout button if user is signed in, otherwise show login */}
+        {user != null ? (
+          <ConfirmLogout user={user} logoutUser={logoutUser} />
+        ) : (
+          <Link href="/login">Log in</Link>
+        )}
+      </div>
+    </>
+  )
+}
+
+// a popover handling user log out
+const ConfirmLogout = (props: {
+  user: CognitoUser
+  logoutUser: (user: CognitoUser) => void
+}) => {
+  const popover = usePopoverState()
+
+  return (
+    <>
+      <PopoverDisclosure {...popover}>Log out</PopoverDisclosure>
+      <Popover {...popover} tabIndex={0}>
+        <div className={logoutPopover}>
+          <div>Log out of DAILP?</div>
+          <Link
+            href=""
```

Comment:
Hmm we have a `Button` component and you should be able to override the styles if that's what you wanted here. I don't remember if we have a pre-built plainer button that is just text. We shouldn't use a `Link` for this.

Reply:
Done, went ahead and used a div styled with the cleanButton instead!

Reply:
If it's semantically a button then the html element should always be `button`. Turning div into a button is also an antipattern unless there's a specific reason.